frontend/src/component/layout/MainLayout/NavigationSidebar/NewInUnleash/NewInUnleashTooltip.tsx

```diff
@@ -0,0 +1,127 @@
+import type * as React from 'react';
+import type { FC, ReactNode } from 'react';
+import { HtmlTooltip } from 'component/common/HtmlTooltip/HtmlTooltip';
+import {
+    Box,
+    Button,
+    Link,
+    styled,
+    Typography,
+    ClickAwayListener,
+} from '@mui/material';
+import { type Link as RouterLink, useNavigate } from 'react-router-dom';
+import OpenInNew from '@mui/icons-material/OpenInNew';
+import { ReactComponent as UnleashLogo } from 'assets/img/logoWithWhiteText.svg';
+
+const Header = styled(Box)(({ theme }) => ({
+    backgroundColor: theme.palette.primary.light,
+    color: theme.palette.primary.contrastText,
+    borderRadius: theme.spacing(1, 1, 0, 0), // has to match the parent tooltip container
+    margin: theme.spacing(-1, -1.5), // has to match the parent tooltip container
+    padding: theme.spacing(2, 3),
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    minHeight: '120px',
+}));
+
+const Body = styled(Box)(({ theme }) => ({
+    padding: theme.spacing(2),
+}));
+
+const StyledLink = styled(Link<typeof RouterLink | 'a'>)(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'center',
+    gap: theme.spacing(1),
+    padding: 0,
+    color: theme.palette.links,
+    fontWeight: theme.fontWeight.medium,
+    '&:hover, &:focus': {
+        textDecoration: 'underline',
+    },
+    '& svg': {
+        fontSize: theme.spacing(2.25),
+    },
```

Comment:
We usually have separate `styled(OpenInNew)` instead of nested selectors. It's easier to debug